.werft/build.ts

```diff
@@ -533,6 +535,21 @@ export async function deployToDevWithInstaller(deploymentConfig: DeploymentConfi
             werft.fail('authProviders', err);
         }
 
+        werft.log("SSH gateway hostkey", "copy host-key from secret")
+        try {
+            exec(`kubectl --namespace keys get secret host-key -o yaml \
```

Comment:
Can you run `kubectl --namespace keys get secret host-key` before https://github.com/gitpod-io/gitpod/blob/c732acb03918e390ae43aa880590eb07e2d4be23/.werft/build.ts#L334 get executed? 

This is needed in the with-vm scenario because we change the kubectl-context awawy from core-dev to the preview environment's k3s cluster.